tools/scripts/fuzz.sh

```diff
@@ -1,21 +1,39 @@
 #!/bin/bash
 
-set -exuo pipefail
+set -euo pipefail
 
-# build the actual fuzz target
-bazel build //test/fuzz:fuzz_dash_e --config=fuzz -c opt
-PATH=$PATH:$(pwd)/bazel-sorbet/external/llvm_toolchain/bin/
-export PATH
-export ASAN_OPTIONS=dedup_token_length=10 # use top 10 frames to tell different errors appart
+what=""
+if [ "$#" -eq 0 ]; then
+  what="fuzz_dash_e"
+elif [ "$#" -eq 1 ]; then
+  what="$1"
+else
+  echo "usage: $0 [<fuzz_target>]"
+  exit 1
+fi
 
+echo "setting env vars"
+export PATH="$PATH:$(pwd)/bazel-sorbet/external/llvm_toolchain/bin"
+export ASAN_OPTIONS='dedup_token_length=10'
 
-# setup corpus
-mkdir -p fuzz_corpus
-find ./test/testdata/ -iname "*.rb"|grep -v disable|xargs -n 1 -I % cp % fuzz_corpus
+echo "checking for commands"
```

Comment:
Background: `llvm-symbolizer` is needed for the fuzzer to symbolize crash stack traces (go from offsets in the Sorbet binary to actual strings for the methods on the stack).

The fuzzer uses the symbolized stack traces to determine if two crashes are "equivalent". I believe it uses the top 10 stack frames.

Reply:
> believe it uses the top 10 stack frames.

This can be changed with an environment variable

Reply:
Do you know what env var it is? Should I see if it's set in this script?

Reply:
It can be configured with an environment variable, but it has a default value. I think it's best to leave it undefined so we can alter it when running `fuzz.sh` if needed.